From 1993f66f586580a360c779cc2d38187f5a96a8a4 Mon Sep 17 00:00:00 2001
From: wjyrich <1071633242@qq.com>
Date: Fri, 23 Aug 2024 13:06:39 +0800
Subject: [PATCH] install

---
 sigviewer.pro | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/sigviewer.pro b/sigviewer.pro
index d73ac51..e5b3027 100644
--- a/sigviewer.pro
+++ b/sigviewer.pro
@@ -79,3 +79,9 @@ SOURCES += \
     $$PWD/src/tab_context.cpp
 
 QMAKE_CFLAGS += -std=c99
+target.path = $$PREFIX/bin
+desktop.files = deploy/debian/sigviewer.desktop
+desktop.path = $$PREFIX/share/applications/
+icons.path = $$PREFIX/share/icons
+icons.files = deploy/debian/sigviewer128.png
+INSTALLS += target desktop icons
\ No newline at end of file
-- 
2.43.4

